src/js/serial_backend.js

```diff
@@ -193,6 +193,12 @@ function finishClose(finishedCallback) {
     // reset active sensor indicators
     sensor_status(0);
 
+    // reset expert mode
+    ConfigStorage.get('permanentExpertMode', function (result) {
+        const checked = result.permanentExpertMode;
+        $('input[name="expertModeCheckbox"]').prop('checked', checked).trigger('change');
+    });
```

Comment:
There is not need to use callback syntax now (we should drop that completely...)
```suggestion
    const result = ConfigStorage.get('permanentExpertMode');
    const checked = result.permanentExpertMode;
    $('input[name="expertModeCheckbox"]').prop('checked', checked).trigger('change');
```

Reply:
@chmelevskij will be fixed in https://github.com/betaflight/betaflight-configurator/pull/2636

Reply:
Ah, though that one is in already.... soz

Reply:
But wait a minute, master already supports sync access, so wouldn't it be better to just go with the correct usage?

Reply:
https://github.com/betaflight/betaflight-configurator/blob/master/src/js/ConfigStorage.js#L32